### PR TITLE
Add start and end date/time into CalendarEvent and all relevant parsers

### PR DESCRIPTION
**Pull request by vig42**
Also added unit/integration tests for the new classes, and update all relevant XML files

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -22,17 +24,19 @@
...
-        + PREFIX_TITLE + "John Doe "
-        + PREFIX_DESCRIPTION + "98765432 "
-        + PREFIX_VENUE + "311, Clementi Ave 2, #02-25 "
-        + PREFIX_TAG + "friends "
-        + PREFIX_TAG + "owesMoney";
+        + PREFIX_TITLE + "CS2103 Final Exam "
+        + PREFIX_DESCRIPTION + "Bring calculator "
+        + PREFIX_VENUE + "MPSH4 "
+        + PREFIX_TAG + "CS2103 "
+        + PREFIX_TAG + "Final";
```
Comment (SleepySanjinLi) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: why this example doesn't have startTime and endTime

File: src/main/java/seedu/address/model/calendarevent/DateTime.java [hunk 2]
```diff
@@ -1,108 +1,123 @@
...
     }
 
     /**
-     * Returns if a given datetime is a valid datetime.
+     * Returns if a given year, month, day, hour and minute correspond to a valid datetime.
      */
     public static boolean isValidDateTime(int year, int month, int day, int hour, int minute) {
-        try {
-            LocalDateTime dateTime = LocalDateTime.of(year, month, day, hour, minute);
-            DateTime dateTime1 = new DateTime(dateTime);
```
Comment (SleepySanjinLi) [inline src/main/java/seedu/address/model/calendarevent/DateTime.java]: in fact, this one can simply check the validation of number.

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (vig42) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: Added in an example for those 2

File: src/main/java/seedu/address/model/calendarevent/DateTime.java (hunk 2 above)
Comment (vig42) [reply src/main/java/seedu/address/model/calendarevent/DateTime.java]: Changed it so that it's using LocalDateTime to validate.

**Pull request merged**